runtime-optenv32/libvpx+32/autobuild/defines

```diff
@@ -1,10 +1,9 @@
 PKGNAME=libvpx+32
-PKGDES="VPx (8, 9, 10) Video Codec SDK (optenv32)"
 PKGSEC=libs
-PKGDEP="glibc+32"
-BUILDDEP="32subsystem"
+PKGDEP="aosc-aaa+32 glibc+32"
+BUILDDEP="devel-base+32 nasm"
```

Comment:
Use `nasm+32` instead?

Reply:
This proves exactly that nasm+32 was not needed. Dropped.